### ISSUE TITLE
Time out for https protocol on the-turing-way.org

### ISSUE THREAD

**Issue by AbcSxyZ** [the-turing-way/the-turing-way#3731]
Related to #3726

The url <http://the-turing-way.org> will work while <https://the-turing-way.org> result in a time out.

**Comment by JimMadge** [comment]
I think this is a limitation of the way NameCheap handles redirects.

The server at `the-turing-way.org` returns a 302 redirect to `https://the-turing-way.start.page`.
It doesn't seem to be accessible over HTTPS but it couldn't work anyway. That server is run by NameCheap, and they don't have a valid cert for `the-turing-way.org`.

The solution is don't prepend those domains with https.

The only way I could see it working is if we ran a server which handled the redirects ourselves.

**Comment by JimMadge** [comment]
Is that something you have done before @jezcope?

If we were managing the webserver ourselves we could get a cert for `*the-turing-way.org` and do something like,

```
server {
    listen       80;
    server_name  the-turing-way.org;
    return 302 https://the-turing-way.start.page;
}
server {
    listen       443;
    server_name  the-turing-way.org;
    return 302 https://the-turing-way.start.page;
}
```

But we don't have that level of control here.

**Comment by jezcope** [comment]
Yup, I use this on https://erambler.co.uk for some `.well-known` URLs and old locations of RSS/Atom feeds: https://codeberg.org/jezcope/erambler/src/branch/production/static/_redirects

But it also supports redirects to any URL, not just one on the same domain.

**Comment by jezcope** [comment]
Or simple cases can go in `netlify.toml` https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

**Comment by JimMadge** [comment]
I think I understand now. The only trick is we will need two sites, one at `the-turing-way.org` for redirects and one at `book.the-turing-way.org` to serve the book.

Not a bad solution, and actually, it would be nice to have all of the redirects defined in plain text in a git repo.

**Comment by jezcope** [comment]
Huh, it might be enough to add `the-turing-way.org` as a domain alias for the book itself, and then have [that domain specifically redirect to the start page](https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects).

**Comment by JimMadge** [comment]
Hmm, yes that sounds like a good route. Should solve similar problems for other redirects that are defined in the DNS provider at the moment.

So the general idea is CNAME records for all of our subdomains point to the site, where we have redirect rules to handle each subdomain.

**Comment by jezcope** [comment]
Ok, I needed to figure this out anyway for another Jupyter Book/Netlify project I'm currently working on, so I've raised an initial PR while it was fresh.
